tests/improver-nbhood/00-null.bats

```diff
@@ -36,7 +36,7 @@
 usage: improver-nbhood [-h]
                        [--radius-in-km RADIUS | --radii-in-km-by-lead-time \
 RADIUS_BY_LEAD_TIME RADIUS_BY_LEAD_TIME]
```

Comment:
I know this is not part of your change but this repeat RADIUS_BY_LEAD_TIME does not make sense

Please can we add 
metavar = ('RADIUS_BY_LEAD_TIME', 'LEAD_TIME_IN_HOURS')
to the CLI and amend these tests appropriately.